wire-tests/build.gradle

```diff
@@ -8,7 +8,7 @@ kotlin {
         kotlin.srcDirs += 'src/jvmKotlinInteropTest/proto-kotlin'
         dependencies {
           implementation(compilations.main.compileDependencyFiles)
-          implementation project(':wire-grpc-client')
+          implementation "com.squareup.wire:wire-grpc-client:3.2.0-SNAPSHOT"
```

Comment:
Should this reference coordinates inside `deps`?

Reply:
yes!